EDDI/EDDI.cs

```diff
@@ -198,6 +187,8 @@ void ActionUpdateSquadronSystem()
                 Task.Run(() =>
                 {
                     // Set up the Frontier API service
+                    // CAUTION: CompanionAppService.Instance must be invoked with the EDDI .ctor to correctly
```

Comment:
My bad, this comment has a typo and should say "within the EDDI .ctor" i.e. it is telling us not to move it into a Task as we are doing here. I think we need to reinstate a line in the .ctor that has the effect of initialising the CompanionAppService singleton before we start spinning off tasks.

Suggestions for re-wording the comment are gladly received!

Reply:
Moved the invocation and and reworded.